### PR TITLE
adding dependabot

### PR DESCRIPTION
**Pull request by eddynaka**
## Changes:
- added dependabot to update only OT dependency. Later we can change that to enable all dependencies if needed.

## Tests:
- GitHub Repo to exemplify: https://github.com/eddynaka/github-dependabot/pulls

### CODE REVIEW

File: .github/dependabot.yml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+version: 2
+updates:
+  # Enable version updates for nuget
+  - package-ecosystem: "nuget"
```
Comment (SergeyKanzhelev) [inline .github/dependabot.yml:4]: will it update from MyGet or from NuGet? Do we want to keep up with the MyGet versions at all?

Comment (eddynaka) [reply .github/dependabot.yml:4]: i really don't know if that would update for myget. i will try in my repo and see if nuget is accepting myget/nuget

Comment (eddynaka) [reply .github/dependabot.yml:4]: @SergeyKanzhelev , with our latest release, i removed the myget feed from this repository, so it won't update from it. Let me know what do you think. Thank you.

Comment (MikeGoldsmith) [reply .github/dependabot.yml:4]: I think having both is good, especially during beta. Nuget is for formally released packages, myget is for bleeding edge daily builds.

Comment (eddynaka) [reply .github/dependabot.yml:4]: we were discussing if we will keep publishing every merge to myget. Probably that will in the next .net SIG

**Pull request closed without merging**